### ISSUE TITLE
Incompatibility issues with werkzeug 1.0

### ISSUE THREAD

**Issue by mcjlnrtwcz** [fengsp/flask-session#112]
On February 6th a new version of Werkzeug was released: https://pypi.org/project/Werkzeug/#history
It introduces a breaking API change which results in a crash. 

### Expected Behavior
Proper version of Werkzeug should be installed.

I think `flask-session` should use fixed version of `flask` - I reported the problem here https://github.com/pallets/flask/issues/3490

### Actual Behavior
Flask application can't be tested.

```pytb
/usr/lib/python3.7/unittest/mock.py:1204: in patched
    return func(*args, **keywargs)
doobar/test_utils/fixtures.py:72: in _stubbed_app
    return create_app('test')
doobar/__init__.py:41: in create_app
    Session(app)
venv/lib/python3.7/site-packages/flask_session/__init__.py:54: in __init__
    self.init_app(app)
venv/lib/python3.7/site-packages/flask_session/__init__.py:61: in init_app
    app.session_interface = self._get_interface(app)
venv/lib/python3.7/site-packages/flask_session/__init__.py:93: in _get_interface
    config['SESSION_USE_SIGNER'], config['SESSION_PERMANENT'])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <flask_session.sessions.FileSystemSessionInterface object at 0x7f38e4d255c0>, cache_dir = '/tmp/doobar-sessions', threshold = 500, mode = 384, key_prefix = 'session:'
use_signer = False, permanent = True

    def __init__(self, cache_dir, threshold, mode, key_prefix,
                 use_signer=False, permanent=True):
>       from werkzeug.contrib.cache import FileSystemCache
E       ModuleNotFoundError: No module named 'werkzeug.contrib'

venv/lib/python3.7/site-packages/flask_session/sessions.py:313: ModuleNotFoundError
```

### Environment

* Python version: 3.7
* Flask version: 1.1.1
* Werkzeug version: 1.0

**Comment by SqrtMinusOne** [comment]
Added a PR to fix this #114

**Comment by charisz** [comment]
Please merge the PR and release, plz, plz, plz.

**Comment by rayluo** [comment]
FYI: I encounter same problem, and I [fixed it in my fork](https://github.com/rayluo/flask-session/pull/1/files), and you can [install it in this way](https://github.com/rayluo/flask-session#how-to-install).

**Comment by VictorSyntez** [comment]
Thank you rayluo!

**Comment by VictorSyntez** [comment]
> FYI: I encounter same problem, and I [fixed it in my fork](https://github.com/rayluo/flask-session/pull/1/files), and you can [install it in this way](https://github.com/rayluo/flask-session#how-to-install).

Thank you!

**Comment by fengsp** [comment]
Fixed since `0.3.2`.
